doc/source/reference/routines.ma.rst

```diff
@@ -80,6 +77,10 @@ Inspecting the array
    ma.size
 
 
+.. seealso:: :ref:`ma.MaskedArray.data <ma-data>`,
+    :ref:`ma.MaskedArray.mask <ma-mask>` and
+    :ref:`ma.MaskedArray.recordmask <ma-recordmask>`
```

Comment:
This seems a little weird. How does it render? Can we inject it into the table with the others somehow?

Reply:
What happens if you use a second `autosummary` without a `toctree` for these?

Reply:
that works better, thanks